kivy/config.py

```diff
@@ -906,7 +906,7 @@ def name(self, value):
             Logger.exception('Core: Error while saving default config file')
 
     # Load configuration from env
-    if environ.get('KIVY_NO_ENV_CONFIG', '0') != '0':
+    if environ.get('KIVY_NO_ENV_CONFIG', '0') == '0':
```

Comment:
It should actually be `if environ.get('KIVY_NO_ENV_CONFIG', '0') != '1':`, because if it's empty it should disable it. Only if it's explicitly `1` should it be disabled.

Reply:
Yes, that's alright. I went the C style if not zero then it's set, but it would allow people to be sloppy (a bit like how it was before by just checking if the variable existing in env instead of checking the value). Cheers.